pkg/dao/dao.go

```diff
@@ -27,7 +23,7 @@ type Dao struct {
 	config    Config
```

Comment:
The etcd client was deliberately private because the Dao is intended to be an abstraction layer on top of the models with controlled public access. I don't think business logic should be talking to etcd directly; it's an encapsulated implementation detail. I'd prefer to keep it private to prevent breaking that design.